code/modules/clothing/suits/armor.dm

```diff
@@ -156,29 +156,37 @@
 	src.item_state = "reactiveoff"
 	..()
 
-/obj/item/clothing/suit/armor/reactive/hit_reaction(mob/living/carbon/human/owner, attack_text, final_block_chance)
+/obj/item/clothing/suit/armor/reactive/teleport
+	name = "reactive teleport armor"
+	desc = "Someone seperated our Research Director from his own head!"
```

Comment:
this should probably have a different description than /obj/item/clothing/suit/armor/reactive